util/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ target_include_directories(util PUBLIC include)
 
 target_link_libraries(util_shared PUBLIC Threads::Threads)
 target_include_directories(util_shared PUBLIC include)
-target_include_directories(util_shared PUBLIC ${CRYPTOPP_INCLUDE_DIRS})
+target_include_directories(util_shared PUBLIC)
```

Comment:
I am not sure I understand it: it seems like we add an empty string to the includes.

Reply:
The whole line is meant to be deleted.